### ISSUE TITLE
Facebook logs me out every app launch

### ISSUE THREAD

**Issue by wolph** [AllanWang/Frost-for-Facebook#1551]
- [x] I have verified that the bug does not occur in my mobile browser (at https://touch.facebook.com/)

**Describe the bug**
Every time after closing the app it asks me for my login again.

**To Reproduce**
Steps to reproduce the behaviour:
1. Open the app
2. Login
3. Use the app
4. Close the app
5. Open the app again, again requesting a login

**Details (please provide at least the app version):**
 - App Version: 2.4.0
 - Device:  Sony XZ2 
 - Android Version: 8.0.0

**Comment by AllanWang** [comment]
Cannot replicate. Is this the first time you are using Frost? It sounds like saving to the database is failing for some reason

**Comment by wolph** [comment]
No, I've been using frost for a while. While debugging I updated to 2.4.0 today (not sure which version I had before, a few weeks old) because I got logged out with the old version.

After getting logged out twice in a row I killed the app and cleared cache and data, no dice however...

**Comment by AllanWang** [comment]
Can you try uninstalling and reinstalling the app? Not just clearing the data. You could also try installing [this test build](https://github.com/AllanWang/Frost-for-Facebook-APK-Builder/releases/tag/v2561), which is the same code but with a different package name.

I did remove migration code form the old database used, but that's existed since 2.3.0 and shouldn't affect you if you are logging in again.

**Comment by wolph** [comment]
That appears to work for now (simply reinstalling).

Liking pages seems to be blocked by Facebook though, I'm getting: "
Sorry, we can't process this request at this time
Sorry, we can't process this request at the moment. Please try again"

But that also occurs at both touch.facebook.com and the regular facebook when visiting from this phone (haven't tried pc yet).

**Comment by AllanWang** [comment]
As in it doesn't work in your browser as well? If it doesn't work there it won't work on Frost. Give some time for issues like these since the mobile site breaks all the time. Given that this  particular issue is resolved. I'm going to close it. You can make a new issue for every repeatable unrelated project

**Comment by wolph** [comment]
Thanks for the quick help! Amazing work on the project :)

**Comment by wolph** [comment]
No luck unfortunately, it took slightly longer but I'm logged out again.

**Comment by wolph** [comment]
It seems that frost (the only recent change in my facebook usage) has caused my account to be flagged:

![image](https://user-images.githubusercontent.com/270571/65750981-e259c500-e109-11e9-8c4f-2eb98401b570.png)

It works sometimes and doesn't work at other times :(

**Comment by AllanWang** [comment]
As far as I'm aware, there is no other reason Frost could be causing phishing problems. This was an ongoing issue for previous builds, and I've rolled back on anything potentially harmful. If this problem is persistent, then you might have to try some other Facebook alternative. If you haven't already, make sure you remove saved devices from facebook before logging in again. Those were the instructions when updating, as the old ones had different user agents

**Comment by wolph** [comment]
I'm currently still running the test build linked above (https://github.com/AllanWang/Frost-for-Facebook-APK-Builder/releases/tag/v2561). It could simply be that my account is still flagged somehow, I should note that I use the chat a lot as well and I remember reading in a different issue that the chat could cause issues.

Just to be sure I've cleared all logins. I didn't have any saved devices to begin with.

Hopefully it'll start working as normal again, otherwise I'll temporarily switch to the mobile site and messenger for the time being. I really do love Frost, it's so much better than the regular app :)

Thank you for all of the help and the great work on this app!
